docker-compose-raft.yml

```diff
@@ -0,0 +1,42 @@
+version: '3.8'
```

Comment:
Makes sense to split this out, since it's basically just for testing

Reply:
I never got the new docker-compose.yml file to work so I could just delete it, but I feel like it's 90% there and we may want to use it at some point